### ISSUE TITLE
Work on missing files

### ISSUE THREAD

**Issue by johndrinkwater** [johndrinkwater/ps-home-archive#2]
https://github.com/AlphaZone4/PSHomeData is a large dataset of Store update files, compare the file listing in https://archive.org/details/psm_home_archive_2015_04_01 with the URLs inside to discover another portion of missing files
Did the work already to pull a 474k list from 2.5GB data… https://gist.github.com/johndrinkwater/dae469baa3d63928a2d88dd8ee5da9fd

**Comment by GTPunk** [comment]
dude, i think its not so necessary since sharc files are unopenable and unusable. and since dudes from AviLife are re-creating exact copies of PSH's spaces. Only thing we can do is collect .bar files from beta, and unpack them for textures.

**Comment by RedFox102** [comment]
@GTPunk I have to agree with you. All the files are encrypted, and it seems that just about everyone has moved on. If there were any fans of PlayStation home that could have decrypted the files into something usable, they would have done it already. Sure, they could keep it to themselves, but what do they have to gain?

There's a reason you only see recreations of popular spaces, it's because no one has been able to get their hands on the original files. I mean it's cool that we have them, but it's not like we can do anything with them.

**Comment by Puterboy1** [comment]
Does anyone have the Slytherin, Ravenclaw and Hufflepuff common rooms from Pottermore? Because I’ve noticed that those scenes are missing.
